docs/UserGuide.adoc

```diff
@@ -207,41 +207,86 @@ exp/23-Nov-2020 a/21`
 
 ==== Add with system prompting
 
-In this mode, the user can add the person's attributes one by one.
-After entering an attribute, the system will prompt the user input for another attribute.
+In this mode, instead of typing a long command, you can add a person's attributes one by one.
+After entering an attribute, the system will prompt you to input another attribute.
 
-Format: `add t/**TYPE**`
+To add a person in this mode, enter the following command of the format: `add t/**TYPE**`
 
-When the user type in the command above, a form will be displayed. For example, if a user wants to add a patient,
-type `add t/patient` to the command box. The form will be as below:
+ORGANice will prompt you to type in each attribute, one by one until all the fields are filled.
+Here are some special commands that might be useful when you are in the process of filling up the form:
 
-image::form.png[width="790"]
+* `/abort` - The command to abandon the current form that you are currently filling.
+* `/undo` - The command to undo the last entry that you entered, this can be useful as sometimes you might make some careless mistakes.
```

Comment:
Perhaps can just say "The command to undo the last entry that you entered, **which is useful when you want to undo any changes made**".